pkgs/os-specific/linux/open-iscsi/default.nix

```
@@ -1,26 +1,26 @@
-{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod }:
+{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod, perl, systemd, pkgconf }:
 stdenv.mkDerivation rec {
   name = "open-iscsi-${version}";
-  version = "2.0-873-${stdenv.lib.substring 0 7 src.rev}";
+  version = "2.0-877-${stdenv.lib.substring 0 7 src.rev}";
+
+  buildInputs = [ automake autoconf libtool gettext utillinux openisns.lib openssl kmod perl systemd pkgconf ];
 
-  buildInputs = [ automake autoconf libtool gettext utillinux openisns.lib openssl kmod ];
-  
   src = fetchFromGitHub {
     owner = "open-iscsi";
     repo = "open-iscsi";
-    rev = "4c1f2d90ef1c73e33d9f1e4ae9c206ffe015a8f9";
-    sha256 = "0h030zk4zih3l8z5662b3kcifdxlakbwwkz1afb7yf0cicds7va8";
+    rev = "0ec3a81158ef32da0a2866f8e2cc2ab02f2e3662";
```

```suggestion
    rev = "${version}";
```
That should work? The version number is a tag in the repository.